docs/src/configuring-zed.md

```diff
@@ -180,6 +180,22 @@ You can also set other OpenType features, like setting `cv01` to `7`:
 The `left_padding` and `right_padding` options define the relative width of the
 left and right padding of the central pane from the workspace when the centered layout mode is activated. Valid values range is from `0` to `0.4`.
 
+## Direnv Integration
+
+- Description: Settings for direnv integration.
```

Comment:
```suggestion
- Description: Settings for [direnv](https://direnv.net/) integration. Requires `direnv` to be installed. `direnv` integration currently only means that the environment variables set by a `direnv` configuration can be used to detect some language servers in `$PATH` instead of installing them.
```